Content.Server/Cargo/CargoOrderDataManager.cs

```diff
@@ -95,5 +96,10 @@ public List<CargoOrderData> GetOrdersFromAccount(int accountId)
                 return null;
             return account.GetOrders();
         }
+
+        public Tuple<int,int> GetCapacity()
```

Comment:
Use strongly-typed model for the method result.

Reply:
Gonna try the named tuple that pjb suggested